pkg/gobuild/gobuild.go

```diff
@@ -0,0 +1,73 @@
+// Package gobuild provides utilities for building programs and tests
+// for the debugging session.
+
+package gobuild
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/go-delve/delve/pkg/config"
+	"github.com/go-delve/delve/pkg/goversion"
+)
+
+// Remove the file at path and issue a warning to stderr if this fails.
+// This can be used to remove the temporary binary generated for the session.
+func Remove(path string) {
+	err := os.Remove(path)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "could not remove %v: %v\n", path, err)
+	}
+}
+
+// OptFlags generates default build flags to turn off optimization and inlining.
+func OptFlags(args []string) []string {
```

Comment:
As far as I can tell this is only called inside gobuild, it doesn't need to be exported.

Reply:
Good point. Fixed.